samples/SocialWeather/Startup.cs

```diff
@@ -35,7 +35,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
 
             app.UseSockets(o => { o.MapEndpoint<SocialWeatherEndPoint>("/weather"); });
-            app.UseStaticFiles();
+            app.UseFileServer();
```

Comment:
This just makes the sample a little more runnable. Right now it 404s when you Ctrl-F5 it. I could have changed the launch URL but this seemed smoother.